glide.yaml

```diff
@@ -55,3 +55,7 @@ import:
   subpackages:
   - unix
 - package: github.com/samalba/dockerclient
+- package: github.com/samuel/go-zookeeper
+  version: 8c6a1d79a44d2742da3660540d1fd43f4ddcfc8c
+  subpackages:
+    - zk
```

Comment:
TravisCI is still giving us this error:

```
docker run --rm --link containerpilot_consul:consul --link containerpilot_etcd:etcd -v /home/travis/gopath/src/github.com/joyent/containerpilot/vendor:/go/src -v /home/travis/gopath/src/github.com/joyent/containerpilot:/cp/src/github.com/joyent/containerpilot -w /cp/src/github.com/joyent/containerpilot containerpilot_build bash ./scripts/unit_test.sh
discovery/zookeeper/zookeeper.go:7:2: cannot find package "github.com/samuel/go-zookeeper/zk" in any of:
	/cp/src/github.com/joyent/containerpilot/vendor/github.com/samuel/go-zookeeper/zk (vendor tree)
	/usr/local/go/src/github.com/samuel/go-zookeeper/zk (from $GOROOT)
	/go/src/github.com/samuel/go-zookeeper/zk (from $GOPATH)
	/cp/src/github.com/samuel/go-zookeeper/zk
make: *** [test] Error 1
```

@justenwalker any thoughts? It looks like maybe we need to regenerate https://github.com/joyent/containerpilot/blob/master/glide.lock and commit it?